### DISCUSSION TITLE
Klaviyo for OpenMage

### DISCUSSION THREAD

**Discussion by m-overlund** [OpenMage/magento-lts#3541] [Extensions & Modules]
Klaviyo, as one of the biggest and most used eCommerce ESP's, should IMHO be supported on OpenMage - as key infrastructure support is essential to OpenMage's succes.

Klaviyo stopped development on the Magento 1 module when it sunset in June 2020, and the module does not support all events/automations etc. 

Some features is properly easy to enhance - send some extra event data to Klaviyo etc.
Watch changes in Magento 2 module etc.

What should we do in cases like this?

I'm guessing OpenMage core isn't meant for supporting these - but there neither seems to be intiatives for these essential systems.

**Comment by fballiano** [comment]
IMHO store owners should pressure those vendors into maintaining the modules, or sponsor somebody to maintain them (if possible), sadly I don't see any other option.

**Reply by m-overlund** [reply]
That's also where I'm at.
In the case of Klaviyo I'm certain they won't update it (i do pressure them though).

But if that's the options, where do one go (in general) to find developers up for the job?
Is it this page? https://www.openmage.org/partners/index.html

**Reply by fballiano** [reply]
that page should be updated, AFAIK most of the companies listed there don't even support openmage and IMHO shouldn't be listed there, but I'm sure that if you post here you'll find somebody

**Reply by pquerner** [reply]
Probably ask in discord. Theres always someone there lurking and if you make your case, and have open DMs I am sure someone will message you.
